src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/ViewDataDictionary.cs

```diff
@@ -23,8 +24,8 @@ public class ViewDataDictionary : IDictionary<string, object>
         /// Initializes a new instance of the <see cref="ViewDataDictionary"/> class.
         /// </summary>
         /// <param name="metadataProvider">
-        /// <see cref = "IModelMetadataProvider" /> instance used to calculate
-        /// <see cref="ViewDataDictionary.ModelMetadata"/> values.
+        /// <see cref="IModelMetadataProvider"/> instance used to create <see cref="ViewFeatures.ModelExplorer"/>
+        /// instances.
```

Comment:
Well, MMP is only used for this purpose. My intent here was to correct the wording and I don't see the extra explanation as a negative.